steering_vectors/steering_vector.py

```diff
@@ -45,7 +45,8 @@ def patch_activations(
         layer_config: Optional[ModelLayerConfig] = None,
         operator: Optional[PatchOperator] = None,
         multiplier: float = 1.0,
-        min_token_index: int = 0,
+        min_token_index: int | None = None,
+        token_indices: List[int] | slice | Tensor | None = None,
```

Comment:
Here as well, need to update the docstring below with this new option

Reply:
Done in [83b0ccf](https://github.com/steering-vectors/steering-vectors/pull/21/commits/83b0ccff5fa9fdedafe08830592102e55fd0b980)